internal/output/output.go

```diff
@@ -73,33 +72,7 @@ func CmdOutputSingular(outputKind string, input []byte, fn PlaintextOutputFn) (s
 		return "", err
 	}
 
-	return outputFromKind(outputKind, "", SingularOutputter{
-		outputFn:     fn,
-		resource:     r,
-		resourceJSON: input,
-	})
-}
-
-// CmdOutputCreate builds a command response based on the flag the user provided and the shape of
-// the input with a successfully created message. The expected shape is a single JSON object.
-func CmdOutputCreate(outputKind string, input []byte, fn PlaintextOutputFn) (string, error) {
```

Comment:
Removed these to have a single `CmdOutput` function so it's simpler for dynamic resource commands.